code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -159,6 +159,34 @@
 	if(delold)
 		qdel(src)
 
+/mob/living/silicon/pai/attackby(obj/item/W as obj, mob/user as mob, params)
+	if(istype(W, /obj/item/stack/nanopaste))
+		var/obj/item/stack/nanopaste/N = W
+		if(stat == DEAD)
+			to_chat(user, "<span class='danger'>\The [src] is beyond help, at this point.</span>")
+		else if(getBruteLoss() || getFireLoss())
+			adjustBruteLoss(-15)
+			adjustFireLoss(-15)
+			updatehealth()
+			N.use(1)
+			user.visible_message("<span class='notice'>[user.name] applied some [W] at [src]'s damaged areas.</span>",\
+				"<span class='notice'>You apply some [W] at [name]'s damaged areas.</span>")
+		else
+			to_chat(user, "<span class='notice'>All [name]'s systems are nominal.</span>")
+		return
+
+	else if(W.force)

```

Comment:
pAIs already take damage from stuff. Just call `..()` if it's *not* nanopaste. No need to double up on the code.